packages/serverless/src/awslambda.ts

```diff
@@ -65,20 +87,6 @@ export function init(options: Sentry.NodeOptions = {}): void {
  */
 function addServerlessEventProcessor(scope: Scope): void {
   scope.addEventProcessor(event => {
-    event.sdk = {
-      ...event.sdk,
-      name: 'sentry.javascript.serverless',
-      integrations: [...((event.sdk && event.sdk.integrations) || []), 'AWSLambda'],
-      packages: [
-        ...((event.sdk && event.sdk.packages) || []),
-        {
-          name: 'npm:@sentry/serverless',
-          version: SDK_VERSION,
-        },
-      ],
-      version: SDK_VERSION,
-    };
-
     addExceptionMechanism(event, {
       handled: false,
     });
```

Comment:
We should probably move all of `addServerlessEventProcessor`, otherwise we'd be applying only some parts of what was originally intended